plutus-errors/exe-next/Main.hs

```diff
@@ -0,0 +1,12 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Main where
+
+import           ErrorCode
+import           Errors
+import           TH.GenCodes
+
+-- | Executable to help developers by returning a currently-unused error code
+main :: IO ()
+main =  putStrLn $ "An error code that is not currently in-use is: "
+                 ++ show (case maximum $(genCodes allErrors) of
```

Comment:
No parenthesized expressions over multiple lines! In this case it seems like you could just show the thing without unwrapping it.